GetStream/stream-chat-swift
# [BW] Turns off the library evolution for StreamChat and StreamChatUI

### 🎯 Goal

Currently our project was built with library evolution turned on, despite the fact we're not yet ready to support module stability within our frameworks.

Please note that the work to provide module stability for our frameworks is currently in progress. Read more [here](https://github.com/GetStream/stream-chat-swift/discussions/1614).

This caused several compiler warnings to be emitted. The goal of this change is to follow `zero warning policy` in our frameworks in order to not clutter the integrator's project.

### 🧪 Testing

1. Open `StreamChat.xcodeproj`
2. Verify there are no compile-time warnings emitted when `StreamChat` and `StreamChatUI` frameworks are built.

### 🎨 Changes

Build setting `BUILD_LIBRARY_FOR_DISTRIBUTION` was turned off in `StreamChat` and `StreamChatUI`.
Uncovered cases in `switch` statements were covered/compile-time warnings were resolved.

### ☑️ Checklist
